common/scripts/babel-conditional-preprocess.js

```diff
@@ -0,0 +1,80 @@
+Object.defineProperty(exports, '__esModule', { value: true });
+
+const helper = require('@babel/helper-plugin-utils');
+const t = require('@babel/types');
+exports.default = helper.declare((_api, opts) => {
+  const {
+    annotations
+  } = opts;
+
+  function Handle(path) {
+    let { node } = path;
+    let removed = false;
+
+    for (const annotation of annotations) {
+      const {
+        match
+      } = annotation;
+
+      if (node.leadingComments && node.leadingComments.length > 0) {
+        for (const comment of node.leadingComments) {
+          if (comment.value.includes(match)) {
```

Comment:
Single `if`?

Reply:
I found a logic leak if we use if - we wanna remove all the rest following comments after the annotation comment, which means removed = true but value doesn't match, reverting this back